test/morse/handlers_test.clj

```diff
@@ -1,33 +1,38 @@
 (ns morse.handlers-test
   (:require [clojure.test :refer :all]
-            [morse.handlers :as h]))
 
+            [morse.handlers :as handlers]))
 
-(defn command-message [command]
+;; aux fns
+
+(defn command-message
```

Comment:
This too, I assume the intention is to always have have the args on the next line, but this is uncalled for.
I haven't seen this anywhere, neither in bbastov's nor in tonsky's guides.

If you want to help with the style, it should be automatable and supported by editors, something like `clj-fmt` would be okay.

Reply:
Sure. Reverted. What editor do you use?

Reply:
Nevermind... I've added a standard gentleman's set of checks to us: `cljfmt`, `eastwood`, and `kibit`.